asana-helpers-markdown.js

```diff
@@ -204,24 +200,22 @@
     await navigator.clipboard.writeText(markdown);
 
     animateButton(ev.srcElement);
-  }
+  };
 
   // Switch between dark and light themes
   function switchTheme(ev) {
     const currentTheme = document.body.classList;
 
-    const targetTheme = (
-      currentTheme.contains('DesignTokenThemeSelectors-theme--darkMode') ||
-      currentTheme.contains('DesignTokenThemeSelectors-theme--systemDarkMode')
-    )
-      ? 'DesignTokenThemeSelectors-theme--lightMode'
-      : 'DesignTokenThemeSelectors-theme--darkMode';
+    const targetTheme =
+      currentTheme.contains('DesignTokenThemeSelectors-theme--darkMode') || currentTheme.contains('DesignTokenThemeSelectors-theme--systemDarkMode')
+        ? 'DesignTokenThemeSelectors-theme--lightMode'
+        : 'DesignTokenThemeSelectors-theme--darkMode';
 
     currentTheme.remove(
       'DesignTokenThemeSelectors-theme--systemDarkMode',
       'DesignTokenThemeSelectors-theme--systemLightMode',
       'DesignTokenThemeSelectors-theme--darkMode',
-      'DesignTokenThemeSelectors-theme--lightMode',
+      'DesignTokenThemeSelectors-theme--lightMode'
```

Comment:
A lot of the changed lines seem to be from some auto-formatting tool. Not really a problem, but I guess we need to add some eslint config to this repo.